src/submittable.js

```diff
@@ -8,26 +8,26 @@ const propNames = ['onEnter', 'onCancel'];
  * focused on an input element calls an onEvent property.
  */
 class Submittable extends React.Component {
-  onKeyDown = event => {
+  onKeyDown = (event) => {
     if (event.target.tagName === 'INPUT') {
-      if (event.keyCode === 13) {
+      if (event.code === 'Enter') {
```

Comment:
event.keyCode is deprecated 
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode